build.gradle

```diff
@@ -63,6 +63,9 @@ configure(projectsWithFlags('java')) {
             implementation "com.fasterxml.jackson.core:jackson-$it"
         }
 
+        // SnakeYAML
+        implementation 'org.yaml:snakeyaml'
```

Comment:
Have you ever thought about using Jackson instead of SnakeYAML?
https://www.baeldung.com/jackson-yaml
https://stackabuse.com/reading-and-writing-yaml-files-in-java-with-snakeyaml
https://stackabuse.com/reading-and-writing-yaml-files-in-java-with-jackson

I think we need to investigate the advantages and disadvantages before we decide to use SnakeYAML. 🤔

Reply:
Oh, I didn't know about jackson-yaml. I'll see. Thank you!!

Reply:
It seems that jackson-dataformat-yaml is also using SnakeYAML internally.
It is a bit confusing that `JsonNode` is returned even when parsing YAML, but it seems convenient to develop by referring to `Jackson.java` by using the `ObjectMapper` interface as it is.
For now, I'll proceed with jackson-dataformat-yaml.

Reply:
@chacham Sorry for the late reply.
I didn't know that Jackson uses SnakeYAML internally. 😅 

> It is a little bit confusing that JsonNode

If so and Jackson uses SnakeYAML anyway, I think it's okay to use SnakeYAML directly unless Jackson provides better usability. 😅 
@ikhoon Do you have any idea about this?

Reply:
I also prefer using `jackson-dataformat-yaml`. If we directly use SnakeYAML, we eventually convert the parsed node from SnakeYAML into JsonNode for reusing other code.